src/oauth2-server/src/Model/Casts/GrantsCast.php

```
@@ -0,0 +1,42 @@
+        if (is_string($value)) {
+            $value = Json::decode($value);
+        }
+
+        if (is_array($value)) {
+            return array_map(fn ($grant) => new Grant($grant), $value);
+        }
+
+        return [];
+    }
+
+    public function set($model, string $key, $value, array $attributes)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$model'. (Unused Code Rules)

(UnusedFormalParameter)